README.md

```diff
@@ -1,6 +1,29 @@
-# Eclipse GLSP Examples [![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)](https://ci.eclipse.org/glsp/job/eclipse-glsp/job/glsp-examples/job/master) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples/&label=npm)](https://ci.eclipse.org/glsp/job/deploy-npm-glsp-examples) [![build-status-server](https://img.shields.io/jenkins/build?jobUrl=https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples/&label=maven)](https://ci.eclipse.org/glsp/job/deploy-m2-glsp-examples)
+# Eclipse GLSP Examples ![build-status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fci.eclipse.org%2Fglsp%2Fjob%2Feclipse-glsp%2Fjob%2Fglsp-examples%2Fjob%2Fmaster%2F)
+
+This repository contains code examples that demonstrates how build diagram editors with the [Graphical Language Server Platform (GLSP)](https://github.com/eclipse-glsp/glsp). The examples are focused on the integration of GLSP editors with the cloud-based [Eclipse Theia IDE](https://github.com/theia-ide/theia) using the [GLSP theia integration](https://github.com/eclipse-glsp/glsp-theia-integration).
+
+Each examples is self-contained and provides both, a example diagram client and its corresponding GLSP server. 
+
+## Prerequisites
+The following libraries/frameworks need to be installed on your system:
+- [Node.js](https://nodejs.org/en/) `>= 12.14.1 AND < 13`
+- [Yarn](https://classic.yarnpkg.com/en/docs/install#debian-stable) `>=1.7.0`
+- [Java](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) `>=11`
+- [Maven](https://maven.apache.org/) `>=3.6.0`
+
+ The examples are heavily interweaved with Eclipse Theia, so please also check the [prerequisites of Theia](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites).
+
+The web-based/client part of the examples has been developed using [Visual Studio Code](https://code.visualstudio.com/) and the server/java part has been developed with the [Eclipse IDE](https://www.eclipse.org/ide/). However, it's of course also possible to use any other IDE or text editor.
+
+## Usage
+- `git clone https://github.com/eclipse-glsp/glsp-examples.git`
+- `cd <any-example-folder>`
+- `cd client && yarn && cd ..` (build client)
+- `cd server && mvn clean install && cd ..` (build server)
+
```

Comment:
Are there some steps missing? If I perform all the steps as described here everything is built but nothing is running and I cannot see anything, so I would not exactly call that "Usage". I assume that we need to start the server and the client somehow. Could you maybe add these steps as well?

From what I understand through trial and error I need to start the server as follows:
- `cd server/org.eclipse.glsp.example.minimal/target && java -jar org.eclipse.glsp.example.minimal-0.0.1-SNAPSHOT-glsp.jar`

and the client like this:
- `cd client/browser-app && yarn start`

but even then it does not work for me. I always get these errors:
![image](https://user-images.githubusercontent.com/19170971/98819575-4da94800-242d-11eb-875a-9dd28ec65327.png)

In the output of the server it seems that the client is not trying to connect to it at all.